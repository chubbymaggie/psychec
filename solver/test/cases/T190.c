
void f()
{
    List list;
    list = 0;
    list = make();
    list->data = 3.14;
}
