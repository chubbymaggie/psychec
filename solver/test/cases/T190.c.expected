/* Forward declarations */
typedef  struct TYPE_3__   TYPE_1__ ;

/* Type definitions */
struct TYPE_3__ {double data; } ;
typedef  TYPE_1__* List ;

/* Variables and functions */
 TYPE_1__* make () ; 

